packages/ui/components/dialog/Dialog.tsx

```diff
@@ -29,11 +31,14 @@ export function Dialog({
   formValues,
   setLoading,
   icon,
+  truncateTitle = true,
```

Comment:
Is it used? 🤔

Reply:
Good catch! Updated to fix that oversight on my end 😅